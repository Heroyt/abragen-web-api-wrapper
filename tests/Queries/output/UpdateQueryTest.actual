[
	'rows' => [
		(object) /* #9612 */ [
			'id' => '1000000120',
			'storecard_id' => '1000000001',
		],
	],
	'name' => 'test',
]
