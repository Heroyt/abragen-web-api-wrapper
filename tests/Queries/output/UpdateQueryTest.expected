[
	'name' => 'test',
	'rows' => [['id' => '1000000120', 'storecard_id' => '1000000001']],
]
